### PR TITLE
[MOSIP-29946] Added workflow to Test mosip-data.

### PR DESCRIPTION
**Pull request by VSIVAKALYAN**

### CODE REVIEW

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
Add a step to check if postgres-init for mosip_master db completed sucessfully.

**Comment by VSIVAKALYAN** [comment]
@syedsalman3753 added the step where it will Check for the Errors in container & exit with code1 if error exists & if not exit with code0 So please Check & approve PR.

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: .github/workflows/push-trigger.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      inputs:
+        message:
+          description: 'Message for manually triggering'
+          required: false
+          default: 'Triggered for Updates'
+          type: string
+    push:
+      branches:
+        - develop
+        - 1.2.0.1
```
Comment (syedsalman3753) [inline .github/workflows/push-trigger.yml:15]: @VSIVAKALYAN 

syntax is not correct please update it

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

**Review by syedsalman3753 — APPROVED** [review]

File: .github/workflows/push-trigger.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    inputs:
+      message:
+        description: 'Message for manually triggering'
+        required: false
+        default: 'Triggered for Updates'
+        type: string
+  push:
+    branches:
+      - develop
+      - 1.2.0.1
```
Comment (ckm007) [inline .github/workflows/push-trigger.yml:15]: add MOSIP* also to push branches.

**Pull request merged**